engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -71,13 +71,15 @@
           + " Please reduce the size by splitting the modification into multiple commands.";
 
   private static final String ERROR_MESSAGE_VARIABLE_SCOPE_NOT_FOUND =
-      "Expected to modify instance of process '%s' but it contains one or more variable instructions"
-          + " with a scope element id that could not be found: '%s'";
+      """
+      Expected to modify instance of process '%s' but it contains one or more variable instructions \
+      with a scope element id that could not be found: '%s'""";
```

Comment:
This change of using text blocks is a suggestion. I want to discuss if you like it or not. As a result, we can add a section to our style guide.

Reply:
I somewhat like the change, especially when the strings become longer and longer.

Getting used to Java's Textblock formatting also takes some time, so when we see it more often it may help us to fully reap the benefits, like explicit new lines, indenting, and such.

Having said that, I think we should discuss it in the team meeting.

I'll accept it at this time nevertheless.

Reply:
I somewhat like the change, especially when the strings become longer and longer.

Getting used to Java's Textblock formatting also takes some time, so when we see it more often it may help us to fully reap the benefits, like explicit new lines, indenting, and such.

Having said that, I think we should discuss it in the team meeting.

I'll accept it at this time nevertheless.